SuperBuild/python_configure_python_launcher.cmake

```diff
@@ -126,6 +126,21 @@ set(PYTHONLAUNCHER_PYTHONPATH_INSTALLED
   "<APPLAUNCHER_DIR>/../${Slicer_BIN_DIR}/Python"
   )
 
+# Add paths necessary to import VTK
+if (APPLE)
+  list(APPEND PYTHONLAUNCHER_PYTHONPATH_INSTALLED
+    "${PYTHONHOME}/lib/Slicer-4.9"
```

Comment:
Slicer_LIB_DIR is set to "lib/Slicer-4.9", so you might use it here.

Reply:
Great, thanks.